local-volume/provisioner/Makefile

```diff
@@ -37,18 +28,26 @@ else
 endif
```

Comment:
we can remove this block, it won't work anymore with every provisioner getting its own set of tags. Here's what I do now: https://github.com/kubernetes-incubator/external-storage/blob/master/gluster/block/Makefile#L18